health/health_log.c

```diff
@@ -145,6 +146,10 @@ inline void health_alarm_log_save(RRDHOST *host, ALARM_ENTRY *ae) {
                             , ae->new_value
                             , ae->old_value
                             , (uint64_t)ae->last_repeat
+
```

Comment:
please, remove this line.

Reply:
Okay! https://github.com/netdata/netdata/pull/10961/commits/a0aa3155ed3afb1530a8423da7758e2de68ae8a7